php/Terminus/Commands/MachineTokensCommand.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Terminus\Commands;
+
+use Terminus;
+use Terminus\Auth;
+use Terminus\Commands\TerminusCommand;
+use Terminus\Models\User;
+use Terminus\Helpers\Input;
+
+/**
+ * Show information for your Pantheon machine tokens
+ */
+class MachineTokensCommand extends TerminusCommand {
+
+  /**
+   * Instantiates object, ensures login
+   */
+  public function __construct() {
+    Auth::ensureLogin();
+    parent::__construct();
+  }
+
+  /**
+   * Show a list of your macnine tokens on Pantheon
+   *
+   * @subcommand list
+   */
+  public function all($args, $assoc_args) {
+    $user        = new User();
+    $this->log()->info('Listing tokens.');
```

Comment:
Our UI convention is to not log statments that are redundant with the explicit command of the user. E.g. if you input `terminus machine-tokens list`, the first line of output shouldn't be `Listing machine tokens`